data_import.md

```diff
@@ -34,4 +34,8 @@
 
 ## Custom Data with R Script
 
-You can create your own custom data sources by writing R Script or using R packages like quantmod for financial data, gtrend for Google Trend, Rfacebook,
+You can create your own custom data sources by writing R Script or using R packages like quantmod for financial data, gtrend for Google Trend, Rfacebook.
+
+## User Defined Plugin
+
```

Comment:
- "(data source definition like input parameters, underlying R function name, etc)". Adding "with" like "(with data source definition ...)" would be easier to understand.
- "if needed." does not seem to be necessary.